### PR TITLE
Exclude generated C files from source distributions

### PR DESCRIPTION
**Pull request by klaussfreire**
* [x] Updated [CHANGELOG file](https://github.com/klaussfreire/clru/blob/master/CHANGELOG.rst)
* [x] Added link at the end of [CHANGELOG file](https://github.com/klaussfreire/clru/blob/master/CHANGELOG.rst)
* [x] Created test for the code changes
* [x] If required, updated the requirements.txt file

### CODE REVIEW

**Review by mauguignard — APPROVED** [review]

File: clru/VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.1.2
+0.1.3
```
Comment (mauguignard) [inline clru/VERSION]: Is `0.1.2` already released? In that case, I think you should update the Changelog entry to the new version

Comment (klaussfreire) [reply clru/VERSION]: Nope, last released version is 0.1.1

Comment (mauguignard) [reply clru/VERSION]: Why the bump to `0.1.3` then? :thinking:

Comment (klaussfreire) [reply clru/VERSION]: Huh.I thought I had reverted that.

**Pull request merged**